frontend/src/Pages/AdminPage.js

```diff
@@ -28,14 +32,30 @@ export default function AdminPage() {
     return wrongAnswers.length
   }
 
+  const howMany = answers => {
+    const users = answers.map(answer => answer.userName)
+    console.log(users)
+    const number = [...new Set(users)]
+    console.log(number)
+    return number.length
+  }
+
   useEffect(() => {
+    getAllUsers(token)
+      .then(response => response.data)
+      .then(data => {
+        const users = data.map(dat => dat.id)
+        setAllUsers(users.length)
+      })
```

Comment:
Why the mapping?
```suggestion
      .then(users => setAllUsers(users.length))
```

Reply:
Without the mapping I got an errors: either objects are not valid as a react child or cannot find undefined... This way it works